addons/account/models/account_move.py

```diff
@@ -4376,7 +4376,11 @@ def _notify_by_email_prepare_rendering_context(self, message, msg_vals=False, mo
             force_email_company=force_email_company, force_email_lang=force_email_lang
         )
         subtitles = [render_context['record'].name]
-        if self.invoice_date_due and self.payment_state not in ('in_payment', 'paid'):
+        if (
+            self.invoice_date_due
+            and self.move_type in self.get_invoice_types(include_receipts=True)
```

Comment:
```suggestion
            and self.is_invoice(include_receipts=True)
```

Reply:
It's getting better and better 😉 
Changed.